README.md

```diff
@@ -32,6 +32,15 @@ https://drive.google.com/drive/folders/1gk90RoNKPqP_lwWpY8cYFQpLThSHtY2m?usp=sha
 Dataset for testing: images with jpg extension and csv file of input/output data: 
 https://drive.google.com/drive/folders/1XGFnvI2nywZ0e6qcOO4kyAmzYbnEYxoU?usp=sharing 
 
+## Get started
+- clone repository
+- upload photos of possible items with masks from drive and put it into `intelligent_placer_lib/data/`. If you want to 
+use yor own data, you could put it (images and masks) into this dir.
+
+for testing:
+- put dataset into `test/data`
+- put dataset specification json into `test/data`. Example of such file already put here.
+
 ## Placer Algorithm
```

Comment:
It seems, that it is necessary to describe the final version of the algorithm. And the process of its previous changes can be traced by improvement points